httpcore/_backends/asyncio.py

```diff
@@ -107,9 +107,9 @@ def get_http_version(self) -> str:
 
         transport = await asyncio.wait_for(
             loop_start_tls(
-                transport=transport,
-                protocol=protocol,
-                sslcontext=ssl_context,
+                transport,
+                protocol,
+                ssl_context,
```

Comment:
Was this causing issues, or can we do without this change?

Reply:
Not absolute on it, just suggesting for the glory of reducing diff size. Feel free to ignore, maybe it's not idiomatic asyncio to pass these as kwargs after all.

Reply:
I think we were not exercising this code path at all in tests. Specifically we didn't (and still aren't until the introducing support for HTTP requests) call `backport_start_tls` and the calling signature is not correct as you can see.

Could we do without this change? I guess, for the specific purposes of this PR, but we'd be adding it immediately after. Not sure if it's worth it?

Reply:
> the calling signature is not correct

Ah, true, I had not seen that. Fair enough :)